plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/context/SecureChannel.java

```diff
@@ -1071,7 +1070,16 @@ private void keepAlive() {
                             .unwrap(apuMessage -> encryptionHandler.decodeMessage(apuMessage))
                             .check(p -> p.getMessage() instanceof OpcuaOpenResponse)
                             .unwrap(p -> (OpcuaOpenResponse) p.getMessage())
-                            .check(p -> p.getRequestId() == transactionId)
+                            .check(p -> {
```

Comment:
locks a bit like debugging code for me so maybe remove this from this PR and add it to another and explain why you think that should be logged on error

Reply:
Removed in 54c0189